opba-db/src/main/resources/migration/migrations/0000-create-table-banks.xml

```diff
@@ -119,7 +124,7 @@
 
     <changeSet id="fill_services" author="mhr">
         <sql dbms="postgresql" stripComments="true">
-            UPDATE opb_bank_profile SET preferred_approach = 'REDIRECT' WHERE true ;
+            UPDATE opb_bank_profile SET preferred_approach = 'REDIRECT' WHERE preferred_approach = '' ;
```

Comment:
Please keep the previous statement for now

Reply:
done